acro/utils.py

```diff
@@ -64,15 +68,35 @@ def finalise_json(filename: str, results: dict) -> None:
     results : dict
         Outputs to write.
     """
-    # convert dataframes to json
+
     outputs: dict = copy.deepcopy(results)
-    for _, output in outputs.items():
+
+    absolute_path = os.path.dirname(os.path.dirname(__file__))
+    relative_path = OUTPUT_DIRECTORY
+    path = os.path.join(absolute_path, relative_path).replace("\\", "/")
+
+    # check if the outputs directory was already created
+    try:
+        os.makedirs(path)
+        print(f"Directory {OUTPUT_DIRECTORY} created successfully")
```

Comment:
Please change the print functions to `logger.info()`. The logger functions should use lazy evaluation so also change the fstring to something like `logger.info("Directory %s created successfully", OUTPUT_DIRECTORY)`